platform/features/layout/src/LayoutController.js

```diff
@@ -293,9 +347,101 @@ define(
          * view configuration.
          */
         LayoutController.prototype.endDrag = function () {
+            this.frameMoved = true;
```

Comment:
Name this `this.dragInProgress`: it's more correct and descriptive-- the code will speak for itself.  "prevent clearing selection while drag is in progress".